### ISSUE TITLE
Exception with Clion 2018.1 EAP 

### ISSUE THREAD

**Issue by mheistermann** [CoatiSoftware/idea-sourcetrail#3]
Hello,
I'm getting the following error when the plugin is just loaded (but not used) :
```
Access is allowed from event dispatch thread only.
 EventQueue.isDispatchThread()=false isDispatchThread()=false Toolkit.getEventQueue()=com.intellij.ide.IdeEventQueue@5375c602 Current thread: Thread[Sourcetrail plugin TCP listener,6,Idea Thread Group] 1704448299 SystemEventQueueThread: Thread[AWT-EventQueue-0 2018.1#CL-181.3986.16 CLion, eap:true, os:Mac OS X 10.13.3, java-version:JetBrains s.r.o 1.8.0_152-release-1136-b16,6,Idea Thread Group] 2065091252

java.lang.Throwable
	at com.intellij.util.ExceptionUtil.currentStackTrace(ExceptionUtil.java:66)
	at com.intellij.psi.impl.DebugUtil.currentStackTrace(DebugUtil.java:477)
	at com.intellij.diagnostic.LogEventException.<init>(LogEventException.java:32)
	at com.intellij.openapi.application.impl.ApplicationImpl.a(ApplicationImpl.java:1096)
	at com.intellij.openapi.application.impl.ApplicationImpl.assertIsDispatchThread(ApplicationImpl.java:1085)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.c(FocusManagerImpl.java:520)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.getFocusOwner(FocusManagerImpl.java:273)
	at com.intellij.ui.messages.JBMacMessages.a(JBMacMessages.java:96)
	at com.intellij.ui.messages.JBMacMessages.showOkMessageDialog(JBMacMessages.java:88)
	at com.intellij.openapi.ui.Messages.showMessageDialog(Messages.java:335)
	at io.sourcetrail.idea.Ping.send(Ping.java:31)
	at io.sourcetrail.idea.TcpListener.run(TcpListener.java:49)
```

**Comment by mlangkabel** [comment]
Hi @mheistermann, thanks for reporting. Can you tell us which version of the Sourcetrail idea plugin you use?

**Comment by mheistermann** [comment]
It's 0.4.7. Just realized that is not up to date anymore, but it appears I cannot update to 0.4.8 yet without building it myself, correct?

**Comment by mlangkabel** [comment]
Yes, we just released 0.4.8 two days ago, which fixes this issue. We also uploaded the plugin build to [https://plugins.jetbrains.com/plugin/9621-idea-sourcetrail](https://plugins.jetbrains.com/plugin/9621-idea-sourcetrail), but its still saying "pending JetBrains approval".
